Doc/library/string.rst

```diff
@@ -415,8 +415,9 @@ error.
 .. versionchanged:: 3.6
    Added the ``'_'`` option (see also :pep:`515`).
 
-*width* is a decimal integer defining the minimum field width.  If not
-specified, then the field width will be determined by the content.
+*width* is a decimal integer defining the minimum total field width,
+including any signs. If not specified, then the field width will be
```

Comment:
Just to be as general as possible (this also needs to cover numeric-base prefixes, commas, underscores, periods, etc.): 

```suggestion
including any prefixes, separators, and other formatting characters. If not specified, then the field width will be
```